api/client/beacon/mock/BUILD.bazel

```
@@ -2,11 +2,8 @@ load("@prysm//tools/go:def.bzl", "go_library")
 
 go_library(
     name = "go_default_library",
```

```suggestion
    name = "go_default_library",
    testonly = True,
```

Please add test only attribute so we are sure this doesn't link into a production binary